tests/isar/services/readers/test_base_reader.py

```diff
@@ -8,37 +8,19 @@
 from isar.services.readers.base_reader import BaseReader
 from robot_interface.models.geometry.pose import Pose
 from robot_interface.models.mission import Task
-from tests.mocks.mission_definition import mock_mission_definition
-from tests.mocks.robot_variables import mock_pose
+from tests.mocks.mission_definition import MockMissionDefinition
+from tests.mocks.pose import MockPose
 from tests.mocks.task import MockTask
-from tests.utilities import Utilities
 
 
 class TestBaseReader:
-    @pytest.mark.parametrize(
-        "location, expected_output",
-        [
-            (Path("./tests/test_data/test_mission_nofile.json"), None),
-            (Path("./tests/test_data/test_mission_working_notasks.json"), dict),
-            (Path("./tests/test_data/test_mission_working.json"), dict),
-            (Path("./tests/test_data/test_mission_not_working.json"), dict),
-            (Path("./tests/test_data/test_json_file.json"), list),
-        ],
-    )
-    def test_read_json(self, location, expected_output):
-        try:
-            content = BaseReader.read_json(location)
-        except Exception:
-            content = None
-        assert Utilities.compare_two_arguments(content, expected_output)
-
     @pytest.mark.parametrize(
         "dataclass_dict, expected_dataclass",
         [
-            (asdict(mock_mission_definition("long_mission")), Mission),
-            (asdict(MockTask.drive_to()), Task),
-            (asdict(MockTask.take_image_in_coordinate_direction()), Task),
-            (asdict(mock_pose()), Pose),
+            (asdict(MockMissionDefinition.default_mission), Mission),
```

Comment:
Does this stil use the "long mission"? Not sure if it needs to

Reply:
I removed the long mission. It was nothing fundamentally different from the default mission. Correct me if the long mission is still needed.

Reply:
Agreed that it should not be needed